### DISCUSSION TITLE
Only if i run so-test command it showing the alerts in the alert dashboard. Urgent!

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#12398] [Unsupported Versions]
Hi, my alert dashboard is empty!
only if i run so-test command it showing the alerts in the dashboard  for only at that particular time after a minute again the dashboard is becoming empty. 
I mean it will show the past alerts and current alert up-to so-test command runes

**Comment by cm-ops** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 Can you give some specifics on your install/grid?

**Reply by sandraimmaculate** [reply]
thanks for the reply! 
I have installed security onion in rocky Linux AWS cloud with 200gb root volume and 4vpcu 2 NIC eth0 is managing and eth1 is monitoring version of security onion is 2.4

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/801bfc27-748c-43c5-b3f0-8b1eec6714f4)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/0daa51fc-6629-4f4f-8271-678c05cfec02)

kindly help

**Reply by sandraimmaculate** [reply]
please do reply

**Reply by cm-ops** [reply]
Do you see traffic on `eth1`?  `sudo tcpdump -nni eth1`

**Reply by sandraimmaculate** [reply]
when i run sudo tcpdump -nni eth1 i can see only one or two traffic if i wait for 5 to 10 mins  but when i run the command sudo tcpdump -nni eth0 for eth0 i can see many traffic within a second.

sudo tcpdump -nni eth1
dropped privs to tcpdump
tcpdump: verbose output suppressed, use -v[v]... for full protocol decode
listening on eth1, link-type EN10MB (Ethernet), snapshot length 262144 bytes
05:01:37.466985 ARP, Request who-has *****  tell ******, length 28
05:02:33.787047 ARP, Request who-has ****** tell ******, length 28
05:03:30.107662 ARP, Request who-has ****** tell ******, length 28
^C
3 packets captured
3 packets received by filter
0 packets dropped by kernel

**Reply by DebianGuru** [reply]
I'm running it onsite, but in a KVM virtual machine.  First, the host adapter had to be put into promiscuous mode and also, virtual NIC was defaulted as a "bridge", which sounded reasonable, but I had to change it to "passthrough" mode.  Probably not exactly the same, but maybe that will point you in the right direction.  Sounds like your NIC isn't truely seeing all traffic.

**Reply by sandraimmaculate** [reply]
what i have to do? to solve this eth0 is seeing all the traffic but not eth1 its very slow if i wait for 5mins to 10 mins i'm able to see 2 to 3 traffic is this is a reason why i'm not able see alerts in alert dashboard until i ran so-test command

**Reply by DebianGuru** [reply]
Since you're running this in a AWS Cloud environment (which I'm totally unfamiliar with) I'd suggest you also check out their documents somewhere under [https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-networking.html](url).  Perhaps they have support also?  You need to let someone know that you need your card in promiscuous mode (so it sees all traffic, not just multicast and broadcast, but unicast also)  Also, in a traditional LAN, the monitoring interface is plugged into a SPAN port on the switch or a network tap.  I'm assuming there is some form of a virtual switch in your virtual network environment?  Perhaps the problem is that your virtual NIC is not attached to any kind of a virtual switch that is NOT setup in a SPAN (mirror) port?  It makes it very difficult to give you a specific answer without being familiar with AWS and also not knowing if you are proficient with these technologies.  Maybe I'm speaking jibberish?

**Reply by sandraimmaculate** [reply]
yea i'm not at all  proficient with these technologies i'm new to these! IDK what to do
should i have to do anything with NIC ?  which i have manually attached!

[https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-networking.html](https://github.com/Security-Onion-Solutions/securityonion/discussions/url) and this page is giving 404 error

**Reply by sandraimmaculate** [reply]
please reply!

**Reply by DebianGuru** [reply]
I can't help you any further.  I have no knowledge of AWS.  I'd be googling for terms like "AWS" and "Promiscuous" and "sniffing", etc. Good luck.

**Reply by cm-ops** [reply]
Have you reviewed https://docs.securityonion.net/en/2.4/cloud-amazon.html and https://docs.aws.amazon.com/vpc/latest/mirroring/what-is-traffic-mirroring.html?

**Reply by sandraimmaculate** [reply]
yes, but i didn't used security onion image manually i have installed the security onion in rocky like, i have added the NIC card like how they have mentioned in document!  with the same subnet which i have used for instance and security group!


but i want to know the alert dashboard is empty because of this reason ? because the second eth1 is collecting traffic very slowly not like eth0 but atleast those  few traffic it has to show in dashboard right. it only showing the alerts only when i run so-test command 
IDK what is the reason!

**Reply by cm-ops** [reply]
You are only seeing ARP requests on your monitoring interface according to you reply above. You need to look at your networking configuration/traffic mirror in AWS.
